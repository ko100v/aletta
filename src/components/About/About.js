import React, { Component } from 'react'
import styles from './About.css'

class About extends Component {

  render() {
    const { heading, bio } = styles

    return (
      <div>
        <h1 className={heading}>Hi, I am Dimitar Kostov</h1>
        <p className={bio}>This section is all about me, yay!</p>
      </div>
    )
  }

}

export default About