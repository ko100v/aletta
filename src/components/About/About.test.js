import React from 'react'

import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

import toJson from 'enzyme-to-json'

import About from './About'

test('About component should render as expected', () => {
  const component = shallow(<About />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})