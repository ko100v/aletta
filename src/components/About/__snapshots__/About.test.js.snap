// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should render as expected 1`] = `
<div>
  <h1
    className="heading"
  >
    Hi, I am Dimitar Kostov
  </h1>
  <p
    className="bio"
  >
    This section is all about me, yay!
  </p>
</div>
`;
